index.html

```diff
@@ -0,0 +1,147 @@
+        secret of all spices, is knowing when to use it. Once you get a spice in
+        your home, you have it forever.
+        <span>
+          Women never throw out spices. Join us and discover!! in December, in
+          Kigali,Rwanda.</span
+        >
+      </p>
+      <h3 id="time">2021.12.03(WED) ~ 05(FRI)</h3>
+      <p id="location">&#64; Kigali Expo ground Kk 321 st</p>
+    </section>
+    <section class="main-program" id="allProgram">
+      <div>
+        <h4>Main Program</h4>
+        <hr class="home-line" />
+      </div>
+      <ul>
+        <li class="program">
+          <img src="images/program_icon_01.png" alt="Lecture" />
+          <h5>Lecture</h5>
+          <p>
+            Lorem ipsum dolor adipisicing elit. Provident incidunt eos laborum
+            quae
+          </p>
+        </li>
+        <li class="program">
+          <img src="images/program_icon_02.png" alt="Exhibition" />
+          <h5>SS Exhibition</h5>
+          <p>
+            Lorem ipsum dolor adipisicing elit. Provident incidunt eos laborum
+            quae
+          </p>
+        </li>
+        <li class="program">
+          <img src="images/program_icon_03.png" alt="Forum" />
+          <h5>Forum</h5>
+          <p>
+            Lorem ipsum dolor adipisicing elit. Provident incidunt eos laborum
+            quae
+          </p>
+        </li>
+        <li class="program">
+          <img src="images/program_icon_04.png" alt="Workshop" />
+          <h5>Workshop</h5>
+          <p>
+            Lorem ipsum dolor adipisicing elit. Provident incidunt eos laborum
+            quae
+          </p>
+        </li>
+        <li class="program">
+          <img src="images/program_icon_05.png" alt="ignite" />
+          <h5>SS Ignite</h5>
+          <p>
+            Lorem ipsum dolor adipisicing elit. Provident incidunt eos laborum
+            quae
+          </p>
+        </li>
+      </ul>
+      <a href="#">SEE THE WHOLE PROGRAM</a>
+      <button id="join-btn" type="button">Join S & S Exhibition 2021</button>
+    </section>
+    <section class="chefs-section"></section>
+    <button class="more-btn" type="button">
+      More
+      <i class="fa fa-chevron-down" aria-hidden="true"></i>
+      <i class="fa fa-chevron-up" aria-hidden="true"></i>
+    </button>
+    <section class="partner">
+      <div>
+        <h4>Partner</h4>
+        <hr class="home-line" />
+      </div>
+      <ul>
+        <li class="partners-logo">
+          <img src="images/world-food-programme-logo.png" alt="WFp" />
+          <img src="images/fao-logo.png" alt="Fao" />
+          <img src="images/Yummy-Spices-logos.jpeg" alt="Yummy spices" />
+        </li>
+      </ul>
+    </section>
+    <footer>
+      <img src="images/logo.jpeg" class="logo" alt="Logo" />
+      <div>
+        <p>Global Spices & Spices Exhibition</p>
+        <p>&#169; 2021. All right deserved.</p>
+      </div>
+    </footer>
```

Comment:
[OPTIONAL]
- In the desktop size: Let's align the footer description to the right to follow the provided design guidelines:

<img width="947" alt="image" src="https://user-images.githubusercontent.com/77190203/152201651-9031b8f9-e380-4924-9359-2983e9897e1d.png">